mods/stairs/init.lua

```diff
@@ -550,6 +651,16 @@ stairs.register_stair_and_slab(
 )
 
 stairs.register_stair_and_slab(
+	"clay",
+	"default:clay",
+	{cracky = 3},
+	{"default_clay.png"},
+	"Clay Stair",
+	"Clay Slab",
+	default.node_sound_stone_defaults()
+)
```

Comment:
Don't worry about it. Not gonna happen anyway.
Maybe someone who is an experienced coder/programmer will do this someday (probably take them roughly ten minutes to do)
I'm not bothering, I'm done with this, enjoy